CONTRIBUTING.md

```diff
@@ -134,6 +135,22 @@ Run the CLI as follows
 consul-k8s 0.36.0-dev
 ```
 
+### Making changes to consul-k8s
+
+The first step to making changes is to fork Consul K8s. Afterwards, the easiest way 
+to work on the fork is to set it as a remote of the Consul K8s project:
+
+1. Rename the existing remote's name: `git remote rename origin upstream`.
```

Comment:
If you fork and clone your fork then you don't need this. I think the workflow should be:
1. Fork
2. Clone your fork
3. Add _our_ repo as `upstream` remote.

Reply:
I think its helpful if you have multiple feature branches for consul-k8s you are working on? I mainly copied and pasted from https://github.com/hashicorp/consul/blob/main/.github/CONTRIBUTING.md#making-changes-to-consul

Reply:
Sure, it's six of one half a dozen of the other.